media-tv/plex-media-server/plex-media-server-1.22.2.4282.ebuild

```diff
@@ -0,0 +1,128 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit pax-utils systemd unpacker
+
+DESCRIPTION="A free media library that is intended for use with a plex client"
+HOMEPAGE="https://www.plex.tv/"
+
+_COMMIT="a97b03fad"
+MY_PV="${PV}-${_COMMIT}"
+
+URI="https://downloads.plex.tv/plex-media-server-new"
+#URI="https://artifacts.plex.tv/plex-media-server-experimental/"
```

Comment:
URI is not a standard variable and looks a bit confusing here. Maybe use MY_URI or MY_BASE_URI?

Reply:
Sound good

Reply:
Updated